src/components/ThreeDS2/ThreeDS2Challenge.tsx

```diff
@@ -2,18 +2,22 @@ import { h } from 'preact';
 import UIElement from '../UIElement';
 import ThreeDS2Challenge from './components/Challenge';
 
-/**
- * ThreeDS2ChallengeElement
- */
-class ThreeDS2ChallengeElement extends UIElement {
+interface ThreeDS2ChallengeElementProps {
+    challengeToken?: string;
+    dataKey?: string;
+    notificationURL?: string;
+    onError?: (error: object | string) => void;
+    paymentData?: string;
+    size?: string;
+    type?: string;
+}
+
+class ThreeDS2ChallengeElement extends UIElement<ThreeDS2ChallengeElementProps> {
     public static type = 'threeDS2Challenge';
 
     public static defaultProps = {
         dataKey: 'threeds2.challengeResult',
-        challengeContainer: null,
         size: '01',
```

Comment:
Why have you removed the `null` default props?

Reply:
Well, does it add any value to set it to `null` instead of just being undefined?
I might be missing something.

Reply:
Just that it adds a clear indication of what all the props will be, once they are defined.
It became quite complicated keeping track of what particular 3DS2 processes were expecting to be sent so this presented a clear list, for reference.

Reply:
Well, we now have `ThreeDS2ChallengeElementProps` to list all the props, right?

Reply:
Ok. But it does miss `challengeContainer`

Reply:
Ah, I see. I'll check this, thanks for pointing it out!

Reply:
It seems `challengeContainer` is not used anymore, so it's fine to remove it.